.github/workflows/esm_tools_actions_novm.yml

```diff
@@ -0,0 +1,66 @@
+name: esm-tools no Docker
+
+# Controls when the action will run. Triggers the workflow on push or pull request.
+
+on: [push, pull_request]
+
+
+jobs:
+  basic_tests_no_docker:
+    runs-on: ubuntu-latest
+    env:
+          LC_ALL: en_US.UTF-8
+          LANG: en_US.UTF-8
+    strategy:
+      matrix:
+        python-version: [ '3.6', '3.7', '3.8', '3.9']
```

Comment:
I think we still have at least one machine using Python 3.5. But we also use f-strings, which aren't in there until 3.6....so, that will fly around the ears at some point.

Reply:
I added 3.5, and it failed with `SyntaxError: invalid syntax` (https://github.com/esm-tools/esm_tools/runs/3242808547?check_suite_focus=true). I will remove now. It was actually a good test showing it's working :)

Reply:
Yup, we get a big red X. I'll adapt the setup.py to ensure it doesn't even try to install itself if you aren't using 3.6 or newer. 

I at some point for one my private projects also had figured out how to get the CI to automatically publish stuff to PyPI. This might also be something to look into